pocketmine-mp/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+---
+version: "3"
+
+services:
+  server:
+    image: pmmp/pocketmine-mp:latest
+    container_name: server
+    restart: always
+    ports:
+      - "19132:19132"
+      - "19132:19132/udp"
```

Comment:
Add comments to explain how to change the port?

Reply:
I considered actually making those available as part of a `server-settings.env` environment file so you can change them from outside the compose file because that might be confusing for some